cmd/activator/main.go

```diff
@@ -64,6 +71,16 @@ const (
 
 	// Add enough buffer to not block request serving on stats collection
 	requestCountingQueueLength = 100
+
+	// The number of requests that are queued on the breaker before the 503s are sent
+	// the value must be adjusted depending on the actual production requirements
+	breakerQueueDepth = 10000
+
+	// The upper bound for concurrent requests sent to the revision,
+	// as new endpoints show up, the Breakers concurrency increases up to this value
+	breakerMaxConcurrency = 1000
+
+	defaultResyncInterval = 10 * time.Hour
```

Comment:
@mattmoor should we factor the default we set (in main.go of the controller) out to a common place to reuse?

Reply:
Maybe a constant in `pkg/reconciler` ot even `pkg/controller` in `knative/pkg` would make sense.

Reply:
could add it in a separate PR - to keep this one from growing even more (already XL 😞).

Reply:
SGTM